lua/damagelogs/shared/lang/simplified_chinese.lua

```diff
@@ -0,0 +1,413 @@
+    Lines = "行",
+    FullDisplay = "全屏显示",
+    SteamIDof = "SteamID 为 ",
+    SteamID = "SteamID",
+    Close = "关闭",
+    ShowDeathScene = "展示死亡场景",
+    ShowDamageInfos = "展示伤害信息",
+    EmptyLogsFilters = "空的日志。请检查您的筛选条件！",
+    EmptyLogs = "空的日志",
+    RoundEnded = "回合结束",
+    DamageInfosBetween = "%s 和 %s 之间的 %s 到 %s 的伤害信息。",
+    PlayerInformation = "玩家信息",
+    SelectDate = "选择日期：",
+    SelectRound = "选择回合：",
+    RoundsBetween = "%s:00 和 %s:00 之间的回合",
+    SelectRoundToLoad = "选择需要加载的回合",
+    PleaseSelectRound = "请选中一个回合！",
+    LoadLogsSelected = "加载选定回合的日志",
+    OldLogs = "旧日志",
+    ShotLogs = "开火日志",
+    ShotEvent = "开火事件",
+    CopySteamID = "复制SteamID64",
+    SpectateAttacker = "观看攻击者",
+    SpectateVictim = "观看受害者",
+    FreeMode = "自由模式",
+    EnableMouse = "注意：按C键启用鼠标。",
+    EnableSlowMotion = "启用慢动作",
+    NoWeapon = "<无武器>",
+    scorpse = "的尸体",
+    TakeAction = "采取行动",
+    HaventReported = "您还没有报告任何人！没有报告可以查看。",
+    RDMConclusion = "结论",
+    ViewPreviousReports = "查看报告",
+    YourReport = "您的报告",
+    PlayerResponse = "玩家的回应",
+    NoLoadedReport = "没有加载的报告",
+    CancelReport = "取消这个报告",
+    ReportCanceled = "报告已取消",
+    AlreadyCanceled = "你已经取消了这个报告",
+    ResponseStatus = "响应状态",
+    Canceled = "已取消",
+    ViewChat = "查看聊天",
+    UpdateNotifications = "启用更新通知",
+    Reported = "已报告",
+    Admin = "管理员",
+    ChatPlayers = "玩家",
+    ChatAdmins = "管理员",
+    ChatDisconnected = "<断开连接>",
+    ChatTitle = "Damagelog的聊天系统",
+    Actions = "行动",
+    AddPlayer = "添加玩家",
+    SelectPlayer = "选择玩家",
+    AddSelected = "添加选定的玩家",
+    ForceStay = "强制留下...",
+    ReleaseChat = "释放...",
+    AllPlayers = "所有玩家",
+    InvalidPlayerChat = "无效的玩家！",
+    CannotLeaveChat = "作为唯一的管理员，你不能离开！",
+    AdminsDisconnectedChat = "所有管理员都断开连接了！聊天已被关闭。",
+    ChatClosedBy = "这个聊天已被 %s 关闭。",
+    AdminLeaveChat = "%s 已离开聊天。",
+    AllPlayersShort = "所有玩家",
+    ForcedNotification = "%s 已强制 %s 留在聊天中",
+    ReleasedNotification = "%s 已将 %s 从聊天中释放",
+    ReportHistory = "报告 %u 历史",
+    ActiveChats = "%u 个活跃的聊天",
+    And = "和",
+    VictimReportedDisconnected = "受害者或被报告的玩家已断开连接！",
+    ChatAlready = "对这个报告已经有一个聊天了！",
+    OpenChatNotification = "%s 已为报告 #%u 打开一个聊天。",
+    AddedChatAdmin = "你已被管理员加入到一个聊天中！",
+    ChatClosed = "%s 已关闭报告 #%u 的聊天。",
+    RDMManagerAuto = "(自动)",
+    ReportHadDNA = "这一轮 %s 有你的DNA！",
+    ReportNoDNA = "这一轮 %s 没有你的DNA。",
+    ReportNoDNAInfo = "无法找到 %s 的 DNA 信息",
+    AnAdministrator = "一个管理员",
+    AdminReportID = "管理员报告 #%u",
+    ChatOpenNoMessage = "(这个管理员报告已直接打开一个聊天)",
+    PendingTop = "待处理",
+    ReportsBottom = "报告",
+    ShowPendingReports = "在 HUD 上显示待处理报告的数量",
+    CreateReport = "创建报告",
+    Donate = "捐赠",
+    EnableSound = "启用通知声音",
+    CurrentRound = "当前回合",
+    StandardReport = "标准报告（作为玩家）",
+    StandardAdminReport = "标准管理员报告",
+    AdvancedAdminReportForce = "高级管理员报告：直接强制被报告的玩家回应",
+    AdvancedAdminReportChat = "高级管理员报告：直接与被报告的玩家打开一个聊天",
+    SlayNextRound = "下一轮添加背锅",
```

Comment:
Is it clear in the Chinese translation that "slay" means the player will be killed?

Reply:
I'm translating this as backstabbing because killing doesn't feel good, if it doesn't work then I'll change it to killing :)

Reply:
Sounds good! Thank you